src/components/messenger/list/index.tsx

```diff
@@ -156,7 +156,9 @@ export class Container extends React.Component<Properties, State> {
   };
 
   isUserAlreadyInConversation = (userId: string) => {
-    return this.props.conversations.find((c) => c.otherMembers[0].userId === userId);
+    return this.props.conversations
+      .filter((conversation) => conversation.otherMembers.length === 1)
```

Comment:
Should use the `isOneOnOne` here. Otherwise a group conversation where everyone who has left except these 2 people could possible be opened and that's not really what the user would want.

Reply:
done